lib/pipeline/webpackProcess.js

```diff
@@ -3,7 +3,16 @@ const fs = require('../core/fs');
 const ProcessWrapper = require('../core/processes/processWrapper');
 const webpack = require('webpack');
 const writeFile = require('util').promisify(require('fs').writeFile);
-const {errorMessage} = require('../utils/utils');
+const {errorMessage, recursiveMerge} = require('../utils/utils');
+
+const defaultPipelineConfig = {
+  typescript: false
+};
```

Comment:
I don't know what is the direction on that but up to now, all the configuration happens in core/config.
Is this something we want to change?

Reply:
You are correct. Since `lib/pipeline` isn't a proper embark module yet (we've got a task for that in Pivotal), I did it this way as an interim implementation. Do you think it's okay, for now, if I leave a comment that it should be handled in core/config once pipeline lives in `lib/modules`?

Reply:
For me it is fine with a TODO :)

Reply:
There's a PR in flight for the pipeline module: https://github.com/embark-framework/embark/pull/962

There'll be a bit of rebasing involved, but if you want you can try your changes on top of that. Otherwise, let me know what you need from `lib/modules/pipeline` :)